src/components/Modal/README.md

```diff
@@ -89,10 +89,13 @@ As of v3.17.0, using `Modal` in an embedded app is deprecated. Support for this
 
 ## Best practices
 
+Use modals when merchants must complete an action before they can continue with the main workflow. Avoid using modals to display complex forms or large amounts of information.
```

Comment:
💯

Reply:
Ahhh I love the `kbd` element and all my monospace default children equally.